helm/values.yaml

```diff
@@ -15,25 +15,33 @@ deployment:
   tolerations: []
   affinity: {}
 
-# configure connaisseur service
+# configure Connaisseur service
 service:
   type: ClusterIP
   port: 443
 
-# configure access to the notary server
+### VALIDATORS ###
+# validators are a set of configurations (hosts, types, public keys) that can be
+# used for validating one or multiple images (or image signatures). They are tied
```

Comment:
Just bc I noticed: If you start a sentence in a comment capitalized, then the first sentence should also be capitalized. That kind of (but not strictly) breaks with all the comment starting lower-case. I also think in other places senteces are not capitalized, not sure though